add maksim (#14)
capro-uio/capro-uio.github.io: content/team/m_slivka/index.en.md

---
title : "Maksim Slivka" # Name
designation : 
  - Neuroimaging data processing
  - Optimization & standardization

>   - Machine learning

Author: ```suggestion
  - Machine learning
  - HPC
id: m_slivka
```

social : # themify icon pack : https://themify.me/themify-icons or fontawesome (https://fontawesome.com/icons)

>   - icon : ti-twitter-alt
>     link : ""

Author: ```suggestion
```

  - icon : ti-github 
    link : "https://github.com/Mak2055"

>   - icon: "fa-brands fa-orcid"
>     link: ""
>   - icon: "fa-brands fa-mastodon"
>     link: ""

Author: ```suggestion
```

---

> MSc in Biomedical Engineering with specialization in Computational Neuroscience, Maksim is particularly skilled in processing MR and EEG data. Being on the front end of science, he is interested in developing and applying novel solutions to scientific problems.

Author: ```suggestion
With a MSc in Biomedical Engineering with specialization in Computational Neuroscience, Maksim is particularly skilled in processing MR and EEG data. Being on the forefront of science, he is interested in developing and applying novel solutions to scientific problems.
```
